build

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
 
-source "$HOME/Applications/emsdk/emsdk_env.sh"
+[ -f "$HOME/Applications/emsdk/emsdk_env.sh" ] && source "$HOME/Applications/emsdk/emsdk_env.sh"
```

Comment:
I was sloppy and left in paths specific to my machine. This can just be removed.